docs/_snippets/addon-backgrounds-disabled.md

```diff
@@ -1,4 +1,4 @@
-```js filename="Button.stories.js|jsx" renderer="common" language="js" tabTitle="globals-api"
+```js filename="Button.stories.js|jsx" renderer="common" language="js" tabTitle="Without story globals"
```

Comment:
@jonniebigodes — We don't have to keep using the `tab-title` format. That was only because we were encoding the tab title as part of the filename before. Now that it's in proper metadata, we can use more friendly strings. 🙂